programs/survival/afk_display.sc

```diff
@@ -0,0 +1,60 @@
+__config() -> {
+    'scope' -> 'global',
+    'stay_loaded' -> true
+};
+
+global_teams = read_file('teams.json', 'json');
+if(global_teams == null,
+    global_teams = write_file('teams.json', 'json', {});
+    global_teams = read_file('teams.json', 'json');
+);
+
+// Change these values to your liking
+global_afk_prefix = '';
+global_afk_suffix = '';
+global_afk_timeout = 180; // seconds
+
+if(scoreboard('afkX') == null, scoreboard_add('afkX'));
+if(scoreboard('afkY') == null, scoreboard_add('afkY'));
+if(scoreboard('afkZ') == null, scoreboard_add('afkZ'));
+if(scoreboard('afkScore') == null, scoreboard_add('afkScore'));
+if(team_list('afk_players') == null,
+    team_add('afk_players');
+    team_property('afk_players', 'color', 'gray');
+    team_property('afk_players', 'prefix', global_afk_prefix);
+    team_property('afk_players', 'suffix', global_afk_suffix);
+);
+
+check_afk() -> (
+    for(player('*'),
+        if(and(
+            scoreboard('afkX', _) == scoreboard('afkX', _, _~'pos':0),
+            scoreboard('afkY', _) == scoreboard('afkY', _, _~'pos':1),
+            scoreboard('afkZ', _) == scoreboard('afkZ', _, _~'pos':2);
+            ),
+            scoreboard('afkScore', _, scoreboard('afkScore', _) + 1),
+            scoreboard('afkScore', _, 0);
+        );
+        if(scoreboard('afkScore', _) >= global_afk_timeout,
+            if(_~'team' == 'afk_players', continue());
```

Comment:
Does `continue()` even do anything in an `if`? From the wiki: 
```
break and continue can only be used inside for, c_for, while, loop, map, filter, reduce as well as Minecraft API block loops, scan and volume functions, while break can be used in first as well.```

Reply:
It's inside a `for` loop.

Reply:
This fragment is inside the `for(player('*'), ...)` loop, so in this context it skips a player it currently iterates.

Reply:
I swear I triple checked for a loop. Lmao I'm blind